e2e/cephfs.go

```diff
@@ -25,42 +25,86 @@ var (
+
+	data, err := replaceNamespaceInTemplate(cephfsDirPath + cephfsProvisionerRBAC)
+	if err != nil {
+		e2elog.Logf("failed to read content from %s %v", cephfsDirPath+cephfsProvisionerRBAC, err)
+	}
+	_, err = framework.RunKubectlInput(data, "--ignore-not-found=true", ns, "delete", "-f", "-")
+	if err != nil {
+		e2elog.Logf("failed to delete provisioner rbac %s %v", cephfsDirPath+cephfsProvisionerRBAC, err)
+	}
+
+	data, err = replaceNamespaceInTemplate(cephfsDirPath + cephfsNodePluginRBAC)
+	if err != nil {
+		e2elog.Logf("failed to read content from %s %v", cephfsDirPath+cephfsNodePluginRBAC, err)
+	}
+	_, err = framework.RunKubectlInput(data, "delete", "--ignore-not-found=true", ns, "-f", "-")
+
+	if err != nil {
+		e2elog.Logf("failed to delete nodeplugin rbac %s %v", cephfsDirPath+cephfsNodePluginRBAC, err)
+	}
+
+	createORDeleteCephfsResouces("create")
 }
 
 func deleteCephfsPlugin() {
-	_, err := framework.RunKubectl("delete", "-f", cephfsDirPath+cephfsProvisioner)
+	createORDeleteCephfsResouces("delete")
+}
+
+func createORDeleteCephfsResouces(action string) {
+	data, err := replaceNamespaceInTemplate(cephfsDirPath + cephfsProvisioner)
 	if err != nil {
-		e2elog.Logf("failed to delete cephfs provisioner %v", err)
+		e2elog.Logf("failed to read content from %s %v", cephfsDirPath+cephfsProvisioner, err)
 	}
-	_, err = framework.RunKubectl("delete", "-f", cephfsDirPath+cephfsProvisionerRBAC)
+	_, err = framework.RunKubectlInput(data, action, ns, "-f", "-")
 	if err != nil {
-		e2elog.Logf("failed to delete cephfs provisioner rbac %v", err)
+		e2elog.Logf("failed to %s cephfs provisioner %v", action, err)
 	}
-	_, err = framework.RunKubectl("delete", "-f", cephfsDirPath+cephfsProvisionerPSP)
+
+	data, err = replaceNamespaceInTemplate(cephfsDirPath + cephfsProvisionerRBAC)
 	if err != nil {
-		e2elog.Logf("failed to delete cephfs provisioner psp %v", err)
+		e2elog.Logf("failed to read content from %s %v", cephfsDirPath+cephfsProvisionerRBAC, err)
 	}
-	_, err = framework.RunKubectl("delete", "-f", cephfsDirPath+cephfsNodePlugin)
+	_, err = framework.RunKubectlInput(data, action, ns, "-f", "-")
 	if err != nil {
-		e2elog.Logf("failed to delete cephfs nodeplugin %v", err)
+		e2elog.Logf("failed to %s cephfs provisioner rbac %v", action, err)
 	}
-	_, err = framework.RunKubectl("delete", "-f", cephfsDirPath+cephfsNodePluginRBAC)
+
+	data, err = replaceNamespaceInTemplate(cephfsDirPath + cephfsProvisionerPSP)
 	if err != nil {
-		e2elog.Logf("failed to delete cephfs nodeplugin rbac %v", err)
+		e2elog.Logf("failed to read content from %s %v", cephfsDirPath+cephfsProvisionerPSP, err)
 	}
-	_, err = framework.RunKubectl("delete", "-f", cephfsDirPath+cephfsNodePluginPSP)
+	_, err = framework.RunKubectlInput(data, action, ns, "-f", "-")
 	if err != nil {
-		e2elog.Logf("failed to delete cephfs nodeplugin psp %v", err)
+		e2elog.Logf("failed to %s cephfs provisioner psp %v", action, err)
```

Comment:
better to expand `psp` in above logs.

Reply:
`psp` is not added in this PR, if required we can do a cleanup in follow up PR